src/actions/FirstOpenActions.tsx

```diff
@@ -27,9 +29,15 @@ export const getFirstOpenInfo = async (): Promise<FirstOpenInfo> => {
       firstOpenText = await firstOpenDisklet.getText(FIRST_OPEN)
       firstOpenInfo = asFirstOpenInfo(JSON.parse(firstOpenText))
       firstOpenInfo.isFirstOpen = 'false'
+
+      if (firstOpenInfo.countryCode == null) {
+        // Not critical if we can't get the country code
+        firstOpenInfo.countryCode = await getCountryCodeByIp().catch(() => undefined)
```

Comment:
Do we want to re-save here, so it's available on the next run?

Reply:
No, then they're stuck as a UK-designated device if they happened to open the app in the UK when this change goes live.